### PR TITLE
Doco arbitrary paths in vox

### PR DESCRIPTION
**Pull request by halloleo**
Added the possibility of arbitrary paths to the help strings in vox activate and
vox remove; also updated the documentation accordingly.

Relates to issue #2935.

### CODE REVIEW

File: docs/python_virtual_environments.rst [hunk 1]
```diff
@@ -40,6 +40,11 @@ To activate an environment, run ``vox activate <envname>``::
 
 Instead of ``activate``, you can call ``workon`` or ``enter``.
 
+If you want to activate an environment which is stored somewhere else (maybe because it was created by another tool) you can pass to ``vox deactivate`` a path to a virtual environment::
```
Comment (astronouth7303) [inline docs/python_virtual_environments.rst]: Typo.

Comment (halloleo) [reply docs/python_virtual_environments.rst]: Thanks @astronouth7303. What *is* the typo? And does it resolve the failing of the CI checks? I am a bit at a loss with the CI errors...

Comment (halloleo) [reply docs/python_virtual_environments.rst]: Thanks @astronouth7303. Fixed it in this branch.

File: news/doco-arbitrary-paths-in-vox.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+Added the possibility of arbitrary paths to the help strings in ``vox activate`` and
```
Comment (gforsyth) [inline news/doco-arbitrary-paths-in-vox.rst]:
```suggestion
* Added the possibility of arbitrary paths to the help strings in ``vox activate`` and
```

**Comment by gforsyth** [comment]
Sorry for the delay in getting to this @halloleo -- the failures are from the invalid RST in the news file.  Needs to have a leading `* ` for each bulleted item

**Comment by halloleo** [comment]
Thanks @gforsyth. Accepted your suggestion, but CI checks still fail. :-(

Can't be that hard to write a news entry in reStructuredText. %-\

File: news/doco-arbitrary-paths-in-vox.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* Added the possibility of arbitrary paths to the help strings in ``vox activate`` and
+``vox remove``; also updated the documentation accordingly.
```
Comment (gforsyth) [inline news/doco-arbitrary-paths-in-vox.rst]:
```suggestion
  ``vox remove``; also updated the documentation accordingly.
```

**Comment by halloleo** [comment]
It gets better. :-) But now flake8 found a style problem:

In vox.py at the code

        activate.add_argument('name', metavar='ENV',
                              help='The environment to activate. ENV can be '+
                                   'either a name from the venvs shown by '+
                                   'vox list or the path to an arbitrary venv')

it complains with a "W504 line break after bin operator". So I guess I will restyle it to break *before* the plus sign, right?

**Comment by gforsyth** [comment]
I should've caught that earlier -- my bad.  Easier way is to use parens to wrap the whole thing, like

```python
In [1]: a = ("The environment to activate. ENV can be " 
   ...:      "either a name from the venvs shown by " 
   ...:      "vox list or the path to an arbitrary venv")
In [2]: print(a)                                                         
The environment to activate. ENV can be either a name from the venvs shown by vox list or the path to an arbitrary venv
```

**Comment by halloleo** [comment]
Thanks for the info, @gforsyth. All checks pass now - finally!

**Comment by gforsyth** [comment]
Thanks a ton, @halloleo !  Yay docs!

**Pull request merged**